components/board/board.go

```diff
@@ -77,12 +77,19 @@ type Board interface {
 // An Analog represents an analog pin that resides on a board.
 type Analog interface {
 	// Read reads off the current value.
-	Read(ctx context.Context, extra map[string]interface{}) (int, error)
+	Read(ctx context.Context, extra map[string]interface{}) (int, AnalogRange, error)
```

Comment:
I can change to returning an`AnalogValue` if you prefer it. I thought that adding a new return would be less breaking since if you still just want the bit value only you can get it and do `_` for the analogrange, but still a breaking change either way so probably doesn't matter that much

Reply:
okay I changed it to one return value, prob better to just have one if theres not a great reason to have 2